allure-multi/action.yml

```diff
@@ -0,0 +1,71 @@
+name: "Allure Multi"
+description: "Generate multiple allure reports from artifacts and publish to gh-pages."
+author: "dholth"
+branding:
+  icon: 'book'
+  color: 'green'
+
+# from actions/checkout
+inputs:
+  repository:
+    description: 'Repository name with owner. For example, actions/checkout'
+    default: ${{ github.repository }}
+  ref:
+    description: >
+      The branch, tag or SHA to checkout. When checking out the repository that
+      triggered a workflow, this defaults to the reference or SHA for that
+      event.  Otherwise, uses the default branch.
+  token:
+    description: >
+      Personal access token (PAT) used to fetch the repository. The PAT is configured
+      with the local git config, which enables your scripts to run authenticated git
+      commands. The post-job step removes the PAT.
+      We recommend using a service account with the least permissions necessary.
+      Also when generating a new PAT, select the least scopes necessary.
+      [Learn more about creating and using encrypted secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)
+    default: ${{ github.token }}
+  pattern:
+    description: >
+      Glob pattern to limit artifact downloads.
+    default: allure-*
+  group:
+    description: >
+      Regex matching common part of multiple-artifacts for a single report.
+    default: allure-(.*)
```

Comment:
this conflicts with the default defined in the Python script, furthermore the default in the script will be unused as long as the default here is defined

Reply:
That's intentional. I tested the script locally, but allure-(.*) would be the most generic if you are not us.